client/core/core.go

```diff
@@ -4987,55 +5092,61 @@ func (c *Core) Trade(pw []byte, form *TradeForm) (*Order, error) {
 		var err error
 		crypter, err = c.encryptionKey(pw)
 		if err != nil {
-			return nil, fmt.Errorf("Trade password error: %w", err)
+			return nil, nil, fmt.Errorf("Trade password error: %w", err)
 		}
 		defer crypter.Close()
```

Comment:
Looks like `crypter.Close()` runs way too soon now that this code is put in the `parseTradeCredentials` helper.

Maybe you tested in a scenario where on-the-fly unlocking the wallet was not needed?

Reply:
> Maybe you tested in a scenario where on-the-fly unlocking the wallet was not needed?

Yeah, the testing wallet was always unlocked.